omnigibson/scenes/interactive_traversable_scene.py

```diff
@@ -170,7 +174,12 @@ def _should_load_object(self, obj_info):
         in_rooms = [in_rooms] if isinstance(in_rooms, str) else in_rooms
 
         # Do not load these object categories (can blacklist building structures as well)
-        not_blacklisted = self.not_load_object_categories is None or category not in self.not_load_object_categories
+        task_relevant_names = set(task_metadata["inst_to_name"].values()) if "inst_to_name" in task_metadata else set()
+        is_task_relevant = name in task_relevant_names or category in ("walls", "floors")
```

Comment:
"walls" and "floors" seem too restrictive. What about "ceilings"? And in our 35 new scenes, it's possible to have a building structure with a category beyond these three?

Reply:
Moving this to a constant & including those too. Thanks.